src/tests/system/tests/test_passkey.py

```diff
@@ -502,3 +502,38 @@ def test_passkey__check_passkey_mapping_token_with_ssh_key_and_passkey(
 
     pam_log = client.fs.read(client.sssd.logs.pam)
     assert "Mapping data found is not passkey related" in pam_log, "String was not found in the logs"
+
+
```

Comment:
As discussed, please add a marker for FIPS here so we can identify cases like this for the future when the test environments are ready.

Reply:
After further discussion, we don't need a marker for fips.